### ISSUE TITLE
Default API keys should be "null" and off by default

### ISSUE THREAD

**Issue by therealcmj** [alondmnt/joplin-plugin-jarvis#33]
In a default install the API keys for OpenAPI and the others seem to have invalid values. By default the model should be set to "None" and API keys should be blanked.

This way the plug-in does not automatically send any data until the user explicitly activates it and provides endpoints and/or API keys.

**Comment by alondmnt** [comment]
Fair point. This (null API keys) was actually the behaviour for a long time, until I changed for security reasons (#28). Since the latter issue was fixed in Joplin recently, I can revert back to null keys. Added to feature requests.
